salalib/shapemap.h

```diff
@@ -565,7 +566,7 @@ class ShapeMap : public PixelBase {
 #define __min(x, y) ((x < y) ? x : y)
 #endif
     //
-    double getSpacing() {
+    const double getSpacing() const {

```

Comment:
I don't think the const on the return value makes a lot of sense - it's a copy anyway and will most likely be ignored by the compiler with a warning.